scripts/ctool

```diff
@@ -965,9 +965,22 @@ git_import() {
 
 
 delete_container() {
-    lxc delete --force "$1"
+    local rc=""
+    for x in 1 2 3 4 5; do
+        # this may fail on ZFS backed systems; let's retry a few
+        # times before reporting error
+        lxc delete --force "$1" 2>/dev/null
+        rc=$?
+        if [ "$rc" = "0" ]; then
+            debug 1 "Successfully deleted container after $x attempt(s)."
+            return $rc
+        fi
```

Comment:
Worth including a `sleep 1`?

Reply:
Yes! Thanks for reminding me about that.